naff/client/client.py

```diff
@@ -33,9 +33,7 @@
 
 import naff.api.events as events
 import naff.client.const as constants
-from naff.api.events import RawGatewayEvent, MessageCreate
-from naff.api.events import processors
-from naff.api.events.internal import Component, BaseEvent
+from naff.api.events import MessageCreate, RawGatewayEvent, processors, Component, BaseEvent
```

Comment:
Bonus points if you refactor all existing references to these to also use `events.`, but I'm happy to merge without that